CommandLine/testing/SimpleTests/math_test.sog/create.edl

```diff
@@ -0,0 +1,23 @@
+float f = (float) -.5;
```

Comment:
does enigm support 3.f ?

Reply:
Just write 3f or 3.0f if you want to be pedantic.